### PR TITLE
Warning when using null value on TextInput in EditPost Issue #7782

### PR DESCRIPTION
**Pull request by winniecarole**
Fixes https://github.com/marmelab/react-admin/issues/7782

## Screenshots

image

### CODE REVIEW

File: docs/InputNullValue.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+const postEditArgument;
+
+if (!postEditArgument && context.snapshot[0][0][0] == 'posts') {
+notify('ra.notification.inputError', {
+type: 'warning',
+messageArgs: { smart_count: 1 },
+undoable: mutationMode === 'undoable',
+});
+}
```
Comment (slax57) [inline docs/InputNullValue.md:20]: What is the purpose of this file? :thinking: 

You did not add any new components so there is no need for a dedicated documentation file.

File: packages/ra-core/src/controller/edit/useEditController.ts [hunk 2]
```diff
@@ -130,29 +130,49 @@ export const useEditController = <
...
                     { id, data },
                     {
                         onSuccess: async (data, variables, context) => {
+                            if (context.snapshot[0][0][0] == 'posts') {
+                                postEditArgument =
+                                    data.title !== ' ' &&
+                                    data.teaser !== ' ' &&
+                                    data.title.trim().length !== 0 &&
+                                    data.teaser.trim().length !== 0;
+                            }
```
Comment (slax57) [inline packages/ra-core/src/controller/edit/useEditController.ts:146]: How can you expect this code to work on any other page that the PostEdit page of the demo? :thinking:

**Review by slax57 — CHANGES_REQUESTED** [review]
Sorry but you're heading to the wrong direction imo with this PR. The fix should be around the TextInput component, and should not be specific to the PostEdit page!

Besides, please do not commit IDE specific files:
- .idea/misc.xml
- .idea/workspace.xml

**Comment by slax57** [comment]
> Please tell me where the textInput component is because I can't find it. If possible the file where I have to make the modification

@winniecarole Sorry I do not know this upfront. Event I would have to do somme digging to find out where to make the modification.

Dealing with form inputs is not easy in react-admin. If you are struggling with this fix don't worry, maybe you can start with easier issues to begin with.

**Comment by WiXSL** [comment]
Shouldn't #8188 make this PR unnecessary?

**Comment by fzaninotto** [comment]
No news for some time, closing.

**Pull request closed without merging**